File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarSobresParaDevolucionOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarSobresParaDevolucionOut : ParametroBaseOut
    {
        public List<Sobre> Sobres { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarSobresParaDevolucionOut.ts
import { Sobre } from "src/app/Components/Sobre/class/sobre";
import { ParametroBaseOut } from './ParametroBaseOut';

export class ListarSobresParaDevolucionOut extends ParametroBaseOut{
    
    private sobres: Sobre[];

    constructor()
    {
        super();
        this.sobres = [];
    }

    public get Sobres(): Sobre[] {
        return this.sobres;
    }
    public set Sobres(value: Sobre[]) {
        this.sobres = value;
    }
}